### PR TITLE
feat(construct-3): add presence

### PR DESCRIPTION
**Pull request by Visual917**
## Description 
A rich presence addon for Construct 3, which is a browser based game engine. 

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
The rich presence shows the project name and version that is currently being worked on, or Idle if the user is Idle. It will be developed further with the requests of the community.
<summary> 

S1

S2

 </summary>

</details>

### CODE REVIEW

**Comment by Visual917** [comment]
Presence validator worked on my initial commit, but gave a 501 error on the second one.

File: websites/C/Construct 3/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		"name": "Mur"
+	},
+	"service": "Construct 3",
+	"description": {
+		"en": "Construct 3 is the worlds best game making software. Make your own game in your browser without coding or with Javascript."
+	},
+	"url": "editor.construct.net",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Z9DRjUb.png",
+	"thumbnail": "https://i.imgur.com/Z9DRjUb.png",
```
Comment (Bas950) [inline websites/C/Construct 3/metadata.json:14]: Logo should be 512x512
Thumbnail should be a promotional banner, or a screenshot of the website homepage.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Could you expand it to show more data/pages? Cause it would probably be denied for low quality at this stage.

File: websites/C/Construct 3/metadata.json (hunk 1 above)
Comment (Visual917) [reply websites/C/Construct 3/metadata.json]: > Logo should be 512x512 Thumbnail should be a promotional banner, or a screenshot of the website homepage.

Fixed it.

File: websites/C/Construct 3/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	"description": {
+		"en": "Construct 3 is the worlds best game making software. Make your own game in your browser without coding or with Javascript."
+	},
+	"url": "editor.construct.net",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/LtxinSm.png",
+	"thumbnail": "https://i.imgur.com/iAj7H7c.png",
+	"color": "#D3E1F1",
+	"category": "games",
+	"tags": ["game", "engine", "construct", "construct3", "scirra"]
```
Comment (theusaf) [inline websites/C/Construct 3/metadata.json:17]: This file is formatted incorrectly. Use `yarn format` to format it.

File: websites/C/Construct 3/presence.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+		element = document.querySelector("#propid2");
+	let version = "";
+	if (element instanceof HTMLInputElement) {
+		const { value: elementValue } = element;
+		version = elementValue;
+	}
+	if (version !== "") version = ` | v:${version}`;
+
+	const presenceData: PresenceData = {
+		largeImageKey: "editor",
```
Comment (theusaf) [inline websites/C/Construct 3/presence.ts:17]: Imgur links are preferred over Discord asset names.

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
Inactive for over a week. Closed.

**Pull request closed without merging**